R/lightswitch.R

```diff
@@ -85,6 +92,26 @@ skip_if_vcr_off <- function() {
   invisible()
 }
 
+vcr_turned_off <- function(error_call = caller_env()) {
+  if (the$light_switch$on) {
+    return(FALSE)
+  }
+
+  if (!the$light_switch$ignore_cassettes) {
+    cli::cli_abort(
+      c(
+        "vcr is turned off.",
+        i = "Use {.fun turn_on} to turn it back on.",
+        i = "Or use {.code turn_off(ignore_cassettes = TRUE)} to cassettes completely."
```

Comment:
```suggestion
        i = "Or use {.code turn_off(ignore_cassettes = TRUE)} to ignore cassettes completely."
```

woops, missing word - my fault